views/index.hbs

```diff
@@ -66,20 +66,19 @@
 
   <div class="row">
     <section class="col-md-4">
-      <h2><a class="natural" href="/jobs">Jobs</a></h2>
-      <ul class="list-unstyled">
-        {{# each jobs}}
-        <li>
-          <a href="/jobs/{{slug}}"><strong>{{title}}</strong> at {{company}}</a>
-        </li>
-        {{/ each}}
-      </ul>
+      <h2>Jobs</h2>
+      <p>
+        We're part of the <a href="https://devedmonton.com/">Edmonton 
+        Dev Society!</a> Check out the #opportunities channel in the Slack 
+        workspace for job postings.
+      </p>
 
       <p>
-        <a href="/jobs">See all jobs...</a>
+        Sign up to the Slack by clicking 
+        <a href="https://devedmonton-invite.herokuapp.com/">here and 
```

Comment:
As of last week, you can now send people to https://slack.devedmonton.com to get an invite instead. ;)

Could we make this into something like:

```
Sign up to the Slack by visiting <a href="https://slack.devedmonton.com">https://slack.devedmonton.com</a> and filling out the form.
```

I'd like to start advertising the new url.